privacyidea/static/components/login/controllers/loginControllers.js

```diff
@@ -216,15 +216,15 @@ angular.module("privacyideaApp")
 
                 // Challenge Response always containes mult_challenge!
                 var multi_challenge = error.detail.multi_challenge;
-                if (multi_challenge.length > 1) {
+                if (multi_challenge.length >= 1) {
                     $scope.challenge_message = gettextCatalog.getString('Please confirm with one of these tokens:');
                     $scope.challenge_multiple_tokens = true;
                 } else {
                     $scope.challenge_message = error.detail.message;
                     $scope.challenge_multiple_tokens = false;
                 }
                 for (var i = 0; i < multi_challenge.length; i++) {
-                    if (multi_challenge.length > 1) {
+                    if (multi_challenge.length >= 1) {
                         $scope.challenge_message = $scope.challenge_message + ' ' + multi_challenge[i].serial;
                     }
                     var attributes = multi_challenge[i].attributes;
```

Comment:
The error seems to be in the line below (line 231).
It probably should be:

    if (attributes === null || attributes.hideResponseInput !== true )

because: if we have attributes, but the attributes do not contain "hideRespsoneInput" explicitly, then we need to display the input field. attributes.hideResponseInput is null, if it is not set.